modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -75,7 +75,7 @@
 public class CellAdapter
     implements Cell, CellEventListener, CellEndpoint
 {
-    private static final Logger _log =
+    private static final Logger logger =
```

Comment:
That my be the case. As you see in our code, we have all  kind of  weird formatting. However, if you want to convince us, then this should be a different commit (in an ideal world, of corse).